cmake/jemalloc.cmake

```diff
@@ -23,6 +23,13 @@ else()
   set(DISABLE_CACHE_OBLIVIOUS "--disable-cache-oblivious")
 endif()
 
+if (NOT DISABLE_JEMALLOC_PROFILING)
```

Comment:
```suggestion
if (ENABLE_JEMALLOC_PROFILING)
```

Reply:
I would like to enable it at compile stage by default.